test/dotnet-msbuild.Tests/GivenDotnetBuildInvocation.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.DotNet.Tools.Build;
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.MSBuild.Tests
+{
+    public class GivenDotnetBuildInvocation
+    {
+        [Theory]
+        [InlineData(new string[] { }, @"exec <msbuildpath> /m /v:m /t:Build /clp:Summary")]
+        [InlineData(new string[] { "-o", "foo" }, @"exec <msbuildpath> /m /v:m /t:Build /p:OutputPath=foo /clp:Summary")]
+        [InlineData(new string[] { "--output", "foo" }, @"exec <msbuildpath> /m /v:m /t:Build /p:OutputPath=foo /clp:Summary")]
+        [InlineData(new string[] { "-o", "foo1 foo2" }, @"exec <msbuildpath> /m /v:m /t:Build ""/p:OutputPath=foo1 foo2"" /clp:Summary")]
+        [InlineData(new string[] { "--no-incremental" }, @"exec <msbuildpath> /m /v:m /t:Rebuild /clp:Summary")]
+        [InlineData(new string[] { "-f", "framework" }, @"exec <msbuildpath> /m /v:m /t:Build /p:TargetFramework=framework /clp:Summary")]
```

Comment:
Can we do something about the `/m /v:m`? these are common to all msbuildforwardingapp users and I'm worried that if we add another then we'll have to consider many tests...